pkg/assetfetcher/asset_test.go

```diff
@@ -106,36 +114,29 @@ func TestAssetPayloadToAssetEventError(t *testing.T) {
 	}
 }
 
-func TestAssetPayloadToAssetEventErrorNeverBeenScanned(t *testing.T) {
+// check if we do indeed receive a asset history with an invalid history
+// AssetPayloadToAssetEvent never should receive an invalid asset history,
+// this test is more so to have coverage, and make sure we do catch this in the event
+func TestAssetPayloadToAssetEventLastScannedError(t *testing.T) {
 	tests := []struct {
 		name  string
 		asset Asset
 	}{
 		{
-			"No LastScanned",
```

Comment:
Suggestion: I think it's a good idea to leave this test case in, just to confirm the behavior of this function if an asset has no history.